pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -132,6 +133,13 @@ workflow ImputationBeagle {
       Int beagle_phase_memory_in_gb = if (CountSamples.nSamples <= 1000) then 22 else ceil(beagle_cpu * 1.5)
       Int beagle_impute_memory_in_gb = if (CountSamples.nSamples <= 1000) then 30 else ceil(beagle_cpu * 4.3)
 
+      # max amount of cpus you can ask for is 96 so at a max of 10k samples we can only ask for 9 cpu a sample.
```

Comment:
this block of additions looks to be a duplicate of right above it

Reply:
🙃

Reply:
i'm really good at merge conflict resolution ok